### PR TITLE
allow for prefixing all measurements

### PR DESCRIPTION
**Pull request by davidbirdsong**
nil value means no prefix, old behavior supported

### CODE REVIEW

File: prometheus.lua [hunk 1]
```diff
@@ -205,10 +205,15 @@ end
 --
```
Comment (knyar) [inline prometheus.lua:207]: Could you please document `prefix` here in the comments?

**Comment by knyar** [comment]
Thanks for this! Could you please also document this functionality in `README`?

**Comment by davidbirdsong** [comment]
Done, I've also removed the implied '_' from the supplied prefix, no need to codify that requirement.

**Comment by knyar** [comment]
Thanks!

**Pull request merged**